easybuild/easyconfigs/g/GSettings-Desktop-Schemas/GSettings-Desktop-Schemas-3.34.0-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'MesonNinja'
+
+name = 'GSettings-Desktop-Schemas'
```

Comment:
@Flamefire it looks like the lowercase name `gsettings-desktop-schemas` is used officially, so let's use that?

Reply:
I was under the impression that recent ECs tend to use Capitalized names and wanted to be consistent with that. I don't mind changing this if I'm mistaken.

Reply:
@boegel What do you say?

Reply:
Comparison example:
- https://gitlab.gnome.org/GNOME/gsettings-desktop-schemas/blob/master/gsettings-desktop-schemas.pc.in "Name: gsettings-desktop-schemas"
- https://gitlab.gnome.org/GNOME/gobject-introspection/blob/master/gobject-introspection-no-export-1.0.pc.in "Name: gobject-introspection"

The latter is capital cased in EB

Reply:
We try to stick to the "official" software name.

In this case, compared https://gitlab.gnome.org/GNOME/gsettings-desktop-schemas/blob/master/README with https://gitlab.gnome.org/GNOME/gobject-introspection/blob/master/README.rst...

Reply:
Done